MAUI/PDF-Viewer/Toolbar.md

```diff
@@ -109,14 +109,6 @@ The names of these toolbars and its description are listed in the following sect
 <td>StickyNoteIconsToolbar</td>
 <td>The toolbar for editing sticky note icons appears at the bottom of the mobile PDF viewer.</td>
```

Comment:
<img width="659" alt="image" src="https://github.com/user-attachments/assets/6db13c37-49d1-44aa-a8b2-2017c53c13ee">

Please mention the reason why it is removed

Reply:
Reason updated